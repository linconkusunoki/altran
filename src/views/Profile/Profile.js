import React from 'react'

import Layout from '../../layout'
import { GnomeProfile } from '../../components'

const Profile = ({ match }) => {
  const { id } = match.params
  return (
    <Layout.Wrapper>
      <Layout.Main>
        <GnomeProfile id={id} />
      </Layout.Main>
      <Layout.Footer />
    </Layout.Wrapper>
  )
}

export default Profile
