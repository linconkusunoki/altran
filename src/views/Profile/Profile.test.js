import React from 'react'
import { render } from '@testing-library/react'
import Profile from './'

it('should render correctly', async () => {
  const { container } = render(<Profile match={{ params: { id: '1' } }} />)
  expect(container.firstChild).toMatchSnapshot()
})
