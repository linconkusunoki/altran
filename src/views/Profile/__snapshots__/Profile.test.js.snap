// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <main
    class="sc-bdVaJa dnnmyP"
  >
    <div
      class="sc-htpNat iiWFAZ"
    >
      <div
        class="sc-bwzfXH cnlxnu"
      >
        <p>
          Loading gnome...
        </p>
      </div>
    </div>
  </main>
  <footer
    class="sc-bxivhb bkAgtj"
  >
    <div
      class="sc-htpNat iiWFAZ"
    >
      Crafted with some
       
      <span
        aria-label="made with love"
        role="img"
      >
        ðŸ§™
      </span>
       
      summoned by Lincon Kusunoki
    </div>
  </footer>
</div>
`;
